lstchain/scripts/lstchain_data_muon_analysis.py

```diff
@@ -74,7 +72,7 @@ def main():
     print("max events: {}".format(args.max_events))
 
     # Camera geometry
-    geom = CameraGeometry.from_name("LSTCam-002")
+    geom = CameraGeometry.from_name("LSTCam-003")
```

Comment:
Is `LSTCam-003` already part of current environment?
ctapipe v0.7 comes with what? ctapipe_resources v0.2.18 ?

Reply:
LSTCam-003 was recently included in ctapipe_io_lst (see [New time and geometry #19](https://github.com/cta-observatory/ctapipe_io_lst/pull/19))